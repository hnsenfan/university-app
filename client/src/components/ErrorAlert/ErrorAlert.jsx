import { PropTypes } from 'prop-types'

import Error from '../Icons/General/Error'

import './ErrorAlert.scss'

const ErrorAlert = (props) => {
  return(
    <div className='container--sm alert-box'>
      <div className='d-flex align-items-center'>
        <Error />
        <label className='text-15'>{props.errMessage}</label>
      </div>
    </div>
  )
}

ErrorAlert.propTypes = {
  errMessage: PropTypes.string
}

export default ErrorAlert