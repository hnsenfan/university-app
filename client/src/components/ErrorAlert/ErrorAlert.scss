@import '/src/modules/styles/core/variables';

.alert-box {
  background-color: lighten(map-get($colors, red), 35%);
  color: map-get($colors, red);
  padding: 8px;
  border-radius: 4px;
  text-align: left;
}