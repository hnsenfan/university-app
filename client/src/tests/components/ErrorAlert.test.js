import { shallow } from 'enzyme'
import ErrorAlert from '../../components/ErrorAlert/ErrorAlert'

test('should test ErrorAlert component', () => {
 const wrapper = shallow(<ErrorAlert />)
 expect(wrapper).toMatchSnapshot()
})