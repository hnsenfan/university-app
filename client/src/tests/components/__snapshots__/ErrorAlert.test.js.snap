// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test ErrorAlert component 1`] = `
<div
  className="container--sm alert-box mt-m"
>
  <div
    className="d-flex align-items-center"
  >
    <Error />
    <label
      className="text-15"
    />
  </div>
</div>
`;
